packages/contracts-bedrock/src/L1/LiquidityMigrator.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { ISharedLockbox } from "./interfaces/ISharedLockbox.sol";
+
+/// @title LiquidityMigrator
+/// @notice A contract to migrate the OptimisPortal's ETH balance to the SharedLockbox. One-time use logic, executed in
+/// a batch of transactions to enable the SharedLockbox interaction within the OptimismPortal.
+contract LiquidityMigrator {
```

Comment:
Im thinking about that maybe we should name the contract in a more specific sense than a generic `LiquidityMigrator`. Maybe `SharedLockboxLiquidityMigrator` or similar. I would like to hear any other opinion

Reply:
I think the current name works, and I prefer it rather than a more verbose option.
But if we want to make more verbose, IMO we'd need to make it very explicit -- like `PortalToLockboxLiquidityMigrator`.
We can open the discussion with architect if you think is necessary ;)

Reply:
Hmm again no strong opinion, so lets leave it like that and see it in other reviews